### PR TITLE
Sliders fix

### PR DESCRIPTION
**Pull request by vlevik**

### CODE REVIEW

File: src/modules/shared/ProductSlider/slick-theme.scss [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+//             background-color: #3b3e4a;
+//         }
+//       }
+
+//       &.slick-active button:before {
+//         background-color: #f1f2f9;
+//       }
+//     }
+//   }
+// }
```
Comment (mpohorenyi) [inline src/modules/shared/ProductSlider/slick-theme.scss]: Why is this commented on?

**Review by mpohorenyi — APPROVED** [review]
Good! 🔥
Fix conflicts.
Why do you store banners in the public folder?

**Comment by vlevik** [comment]
> Good! 🔥 Fix conflicts. Why do you store banners in the public folder?

I did it because they don't load from other folder

File: src/modules/shared/ProductSlider/slick-theme.scss (hunk 1 above)
Comment (vlevik) [reply src/modules/shared/ProductSlider/slick-theme.scss]: > Why is this commented on?

sorry, I just forgot to delete this file 😅 I'll fix it in the next pr :)

File: src/modules/shared/ProductSlider/slick-theme.scss [hunk 2]
```diff
@@ -0,0 +1,132 @@
+@charset "UTF-8";
+@import '../../../styles/blocks/grid';
+@import '../../../styles/utils/variables/colors';
+
+/* Arrows */
```
Comment (dsfreedom) [inline src/modules/shared/ProductSlider/slick-theme.scss]: Other than this file, I think everything is fine

**Review by dsfreedom — APPROVED** [review]
Good job)

**Pull request merged**